docs/addons/cluster-autoscaler.md

```diff
@@ -14,7 +14,7 @@ import * as blueprints from '@aws-quickstart/eks-blueprints';
 
 const app = new cdk.App();
 
-const addOn = new blueprints.addons.ClusterAutoscalerAddOn();
+const addOn = new blueprints.addons.ClusterAutoScalerAddOn();
```

Comment:
What changed here?

Reply:
changed the s in ClusterAutoscalerAddOn to uppercase, since that's how you have to call the addon